examples/circ.rs

```diff
@@ -178,6 +180,7 @@ fn main() {
     };
     let cs = match mode {
         Mode::Opt => opt(cs, vec![Opt::ScalarizeVars, Opt::ConstantFold]),
+        Mode::Fhe => opt(cs, vec![]),
```

Comment:
You may want to scalarize input variables for now (i.e., regard an input `u8[2] x` as two inputs `u8 x.0` and `u8 x.1`). If so, consider merging this case with that of `Mode::Mpc` (see my comment below).